### PR TITLE
Cluster name in mdns config

### PR DESCRIPTION
**Pull request by cybertron**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/387/

File: assets/files/usr/local/bin/coredns.sh [hunk 1]
```diff
@@ -4,13 +4,14 @@ set -e
...
 CLUSTER_DOMAIN="$(awk '/search/ {print $2}' /etc/resolv.conf)"
+read -d '.' -a CLUSTER_NAME <<< $CLUSTER_DOMAIN
 API_VIP="$(dig +noall +answer "api.${CLUSTER_DOMAIN}" | awk '{print $NF}')"
 IFACE_CIDRS="$(ip addr show | grep -v "scope host" | grep -Po 'inet \K[\d.]+/[\d.]+' | xargs)"
 SUBNET_CIDR="$(/usr/local/bin/get_vip_subnet_cidr "$API_VIP" "$IFACE_CIDRS")"
 DNS_VIP="$(/usr/local/bin/nthhost "$SUBNET_CIDR" 2)"
 grep -v "${DNS_VIP}" /etc/resolv.conf | tee /etc/coredns/resolv.conf
 
-COREDNS_IMAGE="quay.io/openshift-metalkube/coredns:latest"
+COREDNS_IMAGE="quay.io/openshift-metalkube/coredns:name-filter"
```
Comment (celebdor) [inline assets/files/usr/local/bin/coredns.sh]: The coredns quay build is broken by coredns-machine-kubernetes. Good that I kept the separate builds :-)
```suggestion
COREDNS_IMAGE="quay.io/openshift-metalkube/coredns-mdns:name-filter"
```

Comment (cybertron) [reply assets/files/usr/local/bin/coredns.sh]: Cool, thanks. That got coredns working for me locally, but there's still an issue with the mdns-publisher config not having the cluster_name. I'll push a new commit when I've figured that out.

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/401/

**Pull request merged**